sklearn/metrics/cluster/_supervised.py

```diff
@@ -335,17 +335,45 @@ def adjusted_rand_score(labels_true, labels_pred):
     Parameters
     ----------
     labels_true : int array, shape = [n_samples]
-        Ground truth class labels to be used as a reference
+        Ground truth class labels to be used as a reference.
 
     labels_pred : array-like of shape (n_samples,)
-        Cluster labels to evaluate
+        Cluster labels to evaluate.
 
     Returns
     -------
     ARI : float
        Similarity score between -0.5 and 1.0. Random labelings have an ARI
        close to 0.0. 1.0 stands for perfect match.
+    
+    See Also
+    --------
+    adjusted_mutual_info_score : Adjusted Mutual Information.
+    """
+    (tn, fp), (fn, tp) = pair_confusion_matrix(labels_true, labels_pred)
```

Comment:
Looks like you left off the `References` and `Examples` sections which exist at and below line 363. They need to be included in the docstring.